src/main/java/org/jenkinsci/plugins/github/config/GitHubPluginConfig.java

```diff
@@ -101,6 +102,14 @@ public void setHookUrl(URL hookUrl) {
         }
     }
 
+    public String getBannedCommitter() {
```

Comment:
OK but how can I use the configuration property into others classes ?
For example in DefaultPushGHEventSubscriber.
As described into http://jenkins-ci.org/pull-request-greeting I add change only for need ot this feature

I didn't see any injection mechanism in this plugin, I just copy things I saw in the plugin.